settings/common.py

```diff
@@ -2,8 +2,12 @@
 
 see: https://docs.djangoproject.com/en/dev/ref/settings/
 """
+# Standard Library
+import os
+
 # Third Party Stuff
 import environ
+from dotenv import load_dotenv
```

Comment:
No need to import `load_dotenv`, when `environ` is already there.

Reply:
No need to have load_dotenv. Everything is already there.

You can use like `env('APP_NAME', default='FAKE')`